pkg/flag/report_flags.go

```diff
@@ -89,7 +89,7 @@ var (
 		Name:       "compliance",
 		ConfigName: "scan.compliance",
 		Value:      "",
-		Usage:      "compliance report to generate (nsa)",
+		Usage:      "compliance report to generate (nsa, cis)",
```

Comment:
wonder if there is a way to override it so it will not show `cis` for `trivy k8s` and `nsa` for  `trivy aws` sub command as it is not supported and throw an error if used by users

Reply:
We could leave the specs out of the help text to make it more generic. 
```suggestion
		Usage:      "compliance report to generate",
```

Reply:
We can override usage like https://github.com/aquasecurity/trivy/pull/3130/commits/10ad755a29b1a51c5788ef3ff3bc2be55292698e, but should the value `cis`? I thought it should be `awscis1.2` and `awscis1.4`. Please correct me if I'm missing something.
BTW, even though we can override the accepted values, do we still need the `aws` prefix? I mean `cis1.2` vs `awscis1.2`.

Reply:
@simar7 Please review my change. Also, do we support CIS 1.4 too? Don't we mention that in the doc? Looks like the current doc mentions CIS 1.2 only.

Reply:
I updated the docs to include 1.4 as well.

As for the prefixes, I think we should include them. CIS publishes many benchmarks and we should be clear that its the AWS version of the CIS benchmarks.

Reply:
>As for the prefixes, I think we should include them. CIS publishes many benchmarks and we should be clear that its the AWS version of the CIS benchmarks.

Got it 👍